Header header

float32[] motor_speed        # motor speed from 0 to 1 (i.e. 0-100%)
